src/actions/index.js

```diff
@@ -70,11 +70,25 @@ export function submitProfile(profile) {
     }
 }
 
+export function editProfile(profile) {
+    return (dispatch, getState) => {
+        const { publicKey } = getState().identity.keypair
+        const state = getState()
+        const hasProfile = Object.values(state.profiles)
+            .filter(profile => profile._pk === state.identity.keypair.publicKey)
+        const txId = hasProfile[0].provenance[hasProfile[0].provenance.length-1]._tx;
```

Comment:
`hasProfile[0].provenance[hasProfile[0].provenance.length-1]`

we're using that a lot... maybe it would make sense to shortcut this in the `asset.js`?
ie

```
asset: {
    ...assetFields : <fixed>
    ...metadataFields: <updateable + latest version: provenance[length-1]>
    provenance
    _txId
    _pk
}